QuestionOne2.cpp

```diff
@@ -1,10 +1,4 @@
-//
-//  QuestionOne2.cpp
-//  Codu
-//
-//  Created by Farida Eid on 5/19/17.
-//  Copyright © 2017 Farida Eid. All rights reserved.
-//
+
 
 #include <iostream>
 #include <unordered_map>
```

Comment:
For some reason I can't comment on the line :(
On lines 22-24 you can use the fact that hashes inserts a new element if the key is not found, to replace the if with just hashTable[c]++. You can use the same on line 30, to avoid checking if present.